north/src/manifest.rs

```diff
@@ -160,15 +218,22 @@ pub enum MountFlag {
 }
 
 #[derive(Clone, Eq, PartialEq, Debug, Serialize, Deserialize)]
+pub struct DirMount {
+    pub host: Option<std::path::PathBuf>,
```

Comment:
Why is this a `Option`? Think a bind mount is not valid without the host dir.

Reply:
Because "data" mounts or "persist" do not have a host path. I used to split bind and data.